pkg/providers/gitlab.go

```diff
@@ -28,20 +28,7 @@ type gitLab struct {
 	tag    string
 }
 
-type gitlabFileInfo struct {
-	url, name, displayName string
-	score                  int
-}
-
-func (g *gitlabFileInfo) String() string {
-	if g.displayName != "" {
-		return g.displayName
-	}
-	return g.name
-}
-
```

Comment:
I imagine these are removed because they are unused?

Reply:
Yes, correct. The `unused` linter kicked in here.